docs/core/setup/install-sdk.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Install .NET Core SDK on Windows, Linux, and macOS
+description: Learn how to install .NET Core on Windows, Linux, and macOS. Discover the dependencies required to develop .NET Core apps.
+author: thraka
+ms.author: adegeo
+ms.date: 11/06/2019
+zone_pivot_groups: operating-systems-set-one
+---
```

Comment:
make sure you have the ms.custom metadata for us to review this article for each vs release like the windows article had